components/budget/filters/PeriodFilter.js

```diff
@@ -72,7 +72,13 @@ const DateRange = ({ from, to }) => {
   if (!from && !to) {
     return <FormattedMessage id="DateRange.All" defaultMessage="All" />;
   } else if (from && to) {
-    return <FormattedDateTimeRange from={from} to={to} />;
+    return (
+      <FormattedMessage
+        id="Date.DateRange"
+        defaultMessage="{dateFrom, date, short}-{dateTo, date, short}"
```

Comment:
No strong opinion here, it looks great, but should we take the opportunity to make this a bit more verbose?
```suggestion
        defaultMessage="{dateFrom, date, short} to {dateTo, date, short}"
```
cc @Betree

Reply:
@kewitz : I agree too, this seems a bit more clearer. It's upto you @Betree; let me know. 😄

Reply:
That's a good idea, and it's internationalized anyway so translators will be able to put whatever works best for their locale